samtranslator/swagger/swagger.py

```diff
@@ -902,7 +902,7 @@ def add_models(self, models):
 
             self.definitions[model_name.lower()] = schema
 
-    def add_resource_policy(self, resource_policy, path, api_id, stage):
```

Comment:
unused parameter (after removing the unused api_id parameter from `_get_method_path_uri_list`), so removed + removed from all calls